commons/src/types/ScheduleEditor.ts

```diff
@@ -43,4 +44,7 @@ export interface Manifest extends NylasManifest {
   show_preview: boolean;
   timezone: string;
   screen_bookings: boolean;
+  max_book_ahead_days: number;
+  min_book_ahead_days: number;
```

Comment:
Noticed these two hadn't previously been added; cc @ozsivanov in case I've missed something obvious and they're elsewhere.